tasks/security_policy_apply/Linux.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: "Unzip patch file"
+  unarchive:
+    src: "{{ security_policy_oracle_artifact }}"
+    dest: "{{ java_path }}{{ java_package }}1.{{ java_major_version }}.0_\
+      {{ java_minor_version }}/jre/lib/security"
+    owner: "root"
+    group: "root"
+    mode: "0755"
+  become: True
+  when:
+    - (java_major_version|int == 8 and java_minor_version|int < 151) or (java_major_version|int < 8)
+
+- name: "Apply patch file"
+  shell: "cp -r {{ java_path }}{{ java_package }}1.{{ java_major_version }}.0_\
```

Comment:
please use _synchronize_ module instead of _shell_ command for recursive copy
https://docs.ansible.com/ansible/latest/modules/synchronize_module.html
if it possible of course

Reply:
It is using sync between ansible and remote host